recipes/redis-plus-plus/all/conanfile.py

```diff
@@ -115,11 +115,8 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "redis++")
-        self.cpp_info.set_property("cmake_target_name", "redis++")
-        self.cpp_info.set_property("pkg_config_name", "redis++")
-        self.cpp_info.components["redis++lib"].set_property("cmake_find_package", "redis++" + "_static" if not self.options.shared else "")
-        self.cpp_info.components["redis++lib"].set_property("cmake_target_name", "redis++" + "_static" if not self.options.shared else "")
-        self.cpp_info.components["redis++lib"].set_property("pkg_config_name", "redis++" + "_static" if not self.options.shared else "")
```

Comment:
I would keep this line to avoid to create 2 pkgconfig files, and remove `self.cpp_info.names["pkg_config"] = "redis++"`

Reply:
Then, we'll have to delete the line below:

```
self.cpp_info.set_property("pkg_config_name", "redis++" + "_static" if not self.options.shared else "")
```
To avoid overwriting the PC file like they're both absolute names.

Reply:
it's fine, we want to override